EpsilonAuxBMS/Tests/AuxStatusHelper.h

```diff
@@ -4,3 +4,7 @@
 AuxStatus auxStatus;
 
 void clearAuxStatus();
+
+void setNominalAuxStatus();
+
+void assertAuxStatusEqual();
```

Comment:
Is there a reason why the parameter is missing in the prototype?

Reply:
cuz I forgot

Reply:
lmao nice